.gitignore

```diff
@@ -5,3 +5,15 @@
 _build/
 _sbuild/
 *.o
+*.a
+
+# temp files
+*.swp
+*.swo
+
+# Built simulators
+c_emulator/riscv_sim_RV32
+c_emulator/riscv_sim_RV64
```

Comment:
There are also RVFI-DII versions

Reply:
I'm just going to revert this file, I shouldn't have included it in the commit.